package.json

```diff
@@ -92,5 +92,11 @@
     "test:unit": "CI=true JEST_JUNIT_OUTPUT=reports/jest-junit.xml react-scripts test --testResultsProcessor ./node_modules/jest-junit --env=jsdom",
     "test:lint": "eslint ./src --max-warnings=0 --format junit -o reports/eslint-junit.xml",
     "eject": "react-scripts eject"
-  }
+  },
+  "browserslist": [
```

Comment:
This is the default suggestion, seems sensible for now, but we can always update.